.gitignore

```diff
@@ -0,0 +1,3 @@
+*.fig
```

Comment:
Could you exclude the gitignore file from the pull request. It's better in my opinion if users use their own customized gitignore file.